packages/vscode-extension/lib/expo_router_v2_plugin.js

```diff
@@ -20,6 +21,18 @@ function useRouterPluginMainHook({ onNavigationChange }) {
   const displayParams = new URLSearchParams(params).toString();
   const displayName = `${pathname}${displayParams ? `?${displayParams}` : ''}`;
 
+  useEffect(() => {
+    if (!store.routeNode) {
+      return;
+    }
+    const routeList = extractNestedRouteList(store.routeNode);
+    if (global.__REACT_DEVTOOLS_GLOBAL_HOOK__?.reactDevtoolsAgent?._bridge) {
+      global.__REACT_DEVTOOLS_GLOBAL_HOOK__.reactDevtoolsAgent._bridge.send("RNIDE_routeListRetrieved", {
+        routes: routeList,
+      });
+    }
+  }, [store.routeNode]);
```

Comment:
it looks like we should extend the navigation plugin abstraction for this instead of acquiring the bridge directly here. Would it make more sense to add another method along `onNavigationChange` that we'd pass to the main navigation plugin hook and then use this method here